src/integrations/magni.ts

```diff
@@ -0,0 +1,23 @@
+import fetch, { Response } from 'node-fetch';
+import { fetchOptions } from '../http';
+
+const magniOrigin = process.env.MAGNI_ORIGIN;
+
+export interface SearchResultFeedback {
+  chunkId: string;
+  feedback: number;
+}
+
+export const postFeedback = async (
```

Comment:
@vpol we need your eyes here to double check

Reply:
Works for me.